.github/workflows/update-flake-lock.yml

```
@@ -0,0 +1,66 @@
+      - name: Check Template Flake
+        id: check_template
+        continue-on-error: true
+        run: |
+          cd template
+          sed -i 's|url = "github:richen604/hydenix"|url = "path:../"|' flake.nix
+          nix flake check
+          git checkout flake.nix
+          rm -f flake.lock
+      
+      - name: Create Pull Request
+        uses: peter-evans/create-pull-request@v5
```

actionlint (1.7.7)

the runner of "peter-evans/create-pull-request@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)